DigitalLearningSolutions.Data/DataServices/UserDataService/UserCentreDetailsDataService.cs

```diff
@@ -135,9 +119,12 @@ FROM UserCentreDetails
                         u.Email,
                         u.EmailVerified,
                         h.EmailVerificationHash,
-                        h.CreatedDate AS EmailVerificationHashCreatedDate
+                        h.CreatedDate AS EmailVerificationHashCreatedDate,
+                        IIF(da.Approved = 0, da.CentreID, NULL) AS CentreIdIfEmailIsForUnapprovedDelegate

```

Comment:
Do you use EmailVerificationDetails anywhere else in the repo?

Reply:
Looks like it's only used in this journey to get the details for the primary/centre email

Reply:
Fair.